conans/cli/commands/inspect.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from conans.cli.command import conan_command, COMMAND_GROUPS
+from conans.cli.output import ConanOutput
+from conans.errors import ConanException
+
+
+def _inspect_json_formatter(data):
+    return json.dumps(data, indent=4)
+
+
+@conan_command(group=COMMAND_GROUPS['creator'], formatters={"json": _inspect_json_formatter})
+def inspect(conan_api, parser, *args, **kwargs):
+    """
+    Returns the specified attribute/s of a conanfile
+    """
+    parser.add_argument("path", help="Path to a folder containing a recipe (conanfile.py)")
+    parser.add_argument("attribute", help="Attribute of the conanfile to read", nargs="+")
```

Comment:
Probably try to align this ``attribute`` with the ``--filter`` argument in ``conan graph info``

Reply:
But a filter is to "discard all" and show only some fields. I'm not sure here we can/want to show a full set of fields from the recipe. Or do we? Which ones?

Reply:
The original ``conan inspect`` listed many fields from the recipe, by default. I think I'd try to align on ``conan graph info`` output?